tests/test_feedstock_io.py

```diff
@@ -191,41 +184,33 @@ def test_copy_file(self):
             with open(filename1, "w", encoding="utf-8", newline="\n") as fh:
                 fh.write(write_text)
 
-            self.assertTrue(os.path.exists(filename1))
-            self.assertFalse(os.path.exists(filename2))
+            assert os.path.exists(filename1)
+            assert not os.path.exists(filename2)
             if repo is not None:
-                self.assertFalse(
-                    list(repo.index.iter_blobs(BlobFilter(filename2)))
-                )
+                assert not list(repo.index.iter_blobs(BlobFilter(filename2)))
 
             fio.copy_file(pathfunc(filename1), pathfunc(filename2))
 
-            self.assertTrue(os.path.exists(filename1))
-            self.assertTrue(os.path.exists(filename2))
+            assert os.path.exists(filename1)
+            assert os.path.exists(filename2)
             if repo is not None:
-                self.assertTrue(
-                    list(repo.index.iter_blobs(BlobFilter(filename2)))
-                )
+                assert list(repo.index.iter_blobs(BlobFilter(filename2)))
 
             read_text = ""
             with open(filename2, encoding="utf-8") as fh:
                 read_text = fh.read()
 
-            self.assertEqual(write_text, read_text)
+            assert write_text == read_text
 
             if repo is not None:
                 blob = next(repo.index.iter_blobs(BlobFilter(filename2)))[1]
                 read_text = blob.data_stream[3].read().decode("utf-8")
 
-                self.assertEqual(write_text, read_text)
+                assert write_text == read_text
 
     def tearDown(self):
```

Comment:
The same comment applies to teardown. This should be part of a test fixture IIUIC.